promhttp/src/promhttp.c

```diff
@@ -19,6 +19,8 @@
 #include "microhttpd.h"
 #include "prom.h"
 
+#define MIMETYPE_TEXT_PLAIN "text/plain"
```

Comment:
`text/plain; version=0.0.4; charset=utf-8` is the correct value for this.

Reply:
Thanks, fixed.